### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by ginabeki**
## Building desktop version

I did the following in this project:-
- Updated font size to match desktop version design
- Used grid layout to align items on the page
- Updated HTML markup to adjust some changes on the desktop version
- Checked for lint errors and fixed the errors
- Ensured the website is responsive

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -296,7 +372,428 @@ textarea:focus {
...
+  }
+
+  .lists li {
+    font-size: 0.9rem;
+  }
+
+  /* Contact section style */
+  .contact {
+    margin-top: 200px;
+  }
```
Comment (levy002) [inline styles.css:740]: - [ ] Kindly make sure to add all background images to the contact section in order to meet the provided design :wink: 

    - ### Your design
   image


   - ### Figma design
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @ginabeki   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -221,81 +272,99 @@ <h3>Skills</h3>
...
-        <div class="icons">
+    <!-- ===================Footer section===================== -->
+
+    <footer class="footer">
+      <div class="footer-wrap">
+        <p class="resume">Get my CV</p>
+        <a href="#">
+          <img class="download-icon" src="images/download.svg" alt=""
+        /></a>
+      </div>
```
[the remark this answers is not in the document]
Comment (ginabeki) [reply index.html:330]: Fixed

File: index.html [hunk 3]
```diff
@@ -221,81 +272,99 @@ <h3>Skills</h3>
...
           name="comment"
           id="message"
           placeholder="Enter your message here..."
           maxlength="500"
           required
         ></textarea>
-        <button class="button" type="submit">Start Collaboration</button>
+        <button class="button button-contact" type="submit">
+          Start Collaboration
+        </button>
```
[the remark this answers is not in the document]
Comment (ginabeki) [reply index.html:319]: Fixed

File: index.html [hunk 4]
```diff
@@ -221,81 +272,99 @@ <h3>Skills</h3>
...
     <!-- =============End About Section===================== -->
     <!-- Form section -->
     <section class="contact">
-      <h1>Getting Started</h1>
-      <p>
+      <h1 class="contact-header">Get Started</h1>
+      <p class="contact-text">
         I'm always interested in hearing about new projects, so if you'd like to
         chat please get in touch.
       </p>
```
[the remark this answers is not in the document]
Comment (ginabeki) [reply index.html:290]: Doned

**Comment by levy002** [comment]
Thank you @ginabeki for considering all suggested changes. Kindly feel free to go ahead and submit your PR for another review. :wink:

File: styles.css (hunk 1 above)
Comment (ginabeki) [reply styles.css:740]: Done

**Comment by ginabeki** [comment]
## Revision
Here are some of the things I added in regards to the previous feedback:-
- Added the required text color to the footer section
- Changed the width of the buttons to match the Figma design
- Changed the typography of the contact section paragraph to match the design.
- Added a missing background image to the contact section


#### Thank you for taking your time to review my work 🙌

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @ginabeki    👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors.
- ✔good design 🔥🔥
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**